README.md

```diff
@@ -55,6 +55,7 @@ RDM integrates tightly into modern software development workflows.  Essentially,
 - PyYAML
 - Pandoc and Latex (optional, required for PDFs)
 - Reportlab and Svglib (optional, required to include SVGs in PDFs)
+- GitPython (optional, required to test hooks)
```

Comment:
I would split this out into a separate "Development Dependencies" section, or even just remove it.

We don't want potential users to be scared by the long dependency list.  And, since only developers working on `rdm` itself will need `GitPython` we can safely remove it from here.